src/python/grpcio/grpc/__init__.py

```diff
@@ -721,6 +768,26 @@ def peer(self):
     raise NotImplementedError()
 
   @abc.abstractmethod
+  def parent(self, child=None, census=None):
+    """Creates a value to associate invoked RPCs with the current RPC.
+
+    This method's parameters have no guaranteed defaults; if either True or
+    False is desired for the value of an argument the argument should be
+    explicitly passed.
+
+    Args:
+      child: An optional boolean indicating whether or not the invoked RPC
```

Comment:
What census information gets propagated when child=False?  If it is possible to propagate census data as a root, I think it would be more natural to have some client side API that exposes that.